cmd/parseUrl.go

```
@@ -0,0 +1,72 @@
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/FMotalleb/junction/singbox"
+	"github.com/pelletier/go-toml"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+)
+
+// parseUrlCmd represents the parseUrl command.
+var parseUrlCmd = &cobra.Command{
```

golangci-lint (1.64.8)

[warning] var-naming: var parseUrlCmd should be parseURLCmd

(revive)